[enterprise-4.13] OCPBUGS-19453: add admonition about node reboot (#73711)
openshift/openshift-docs: modules/customize-certificates-replace-default-router.adoc

root CA certificate.
* Copy the root CA certificate into an additional PEM format file.

[IMPORTANT]
====

> Updating the certificate authority (CA) will cause the nodes of your cluster to reboot.

Reviewer: Suggest active voice

```suggestion
Updating the certificate authority (CA) causes the nodes in your cluster to reboot.
```
